README.md

```diff
@@ -8,10 +8,24 @@ I have the time in milliseconds and I want it to become "30 minutes" or "3 days,
 
 **This library is actively maintained but no new features will be added.**
 
+
+## Installation
```

Comment:
Nitpick: could you make sure this heading matches the others?

```diff
-## Installation
+Installation
+------------
```

Reply:
Sorry, I forgot this. Solved now!